deploy/hub/placements.crd.yaml

```diff
@@ -262,6 +262,13 @@ spec:
                   ManagedClusters
                 type: integer
                 format: int32
+    additionalPrinterColumns:
+    - name: Succeeded
+      type: string
+      jsonPath: ".status.conditions[?(@.type==\"PlacementSatisfied\")].status"
```

Comment:
I think you would also  add  the number of selected clusters.

Reply:
Do you means this: @qiujian16 
```
NAME         SUCCEEDED   REASON                     SELECTEDCLUSTERS
placement1   False       NotAllDecisionsScheduled   2
```

Reply:
yes, but I think the  PR should be made in api  and sync to here.